### PR TITLE
Remove deprecated package

### PR DESCRIPTION
**Pull request by Tasiobg**
Similar to the one in frontend https://github.com/bbc/programmes-frontend/commit/846c820e26e2b3b8a2bd5e0db1d40a43d92f7fc7

I have run:
- composer remove symfony/lts
- composer update symfony/flex
- composer config extra.symfony.require "^4.1.0"

I think this is what is required.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: composer.json [hunk 1]
```diff
@@ -105,7 +104,8 @@
         "public-dir": "web",
         "symfony": {
             "id": "01BXQ63SY7AN2CGC7144SGPBHR",
-            "allow-contrib": false
+            "allow-contrib": false,
+            "require": "^4.1"
```
[the remark this answers is not in the document]
Comment (Tasiobg) [reply composer.json]: I copied the version I saw here `"symfony/framework-bundle": "^4.1",`
I don't really know why we don't only update patch versions. Any idea @votemike @benolot ?
I'm not too familiar with symfony installation

[the remark this answers is not in the document]
Comment (votemike) [reply composer.json]: `^4.1.0` looks to be the string we want. It should allow >=4.1.0 <4.2.0 https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

[the remark this answers is not in the document]
Comment (Tasiobg) [reply composer.json]: Cool, but what about what is not beign changed in this PR?
Should we care about `"symfony/framework-bundle": "^4.1"` or others?

[the remark this answers is not in the document]
Comment (votemike) [reply composer.json]: Not sure what you mean

[the remark this answers is not in the document]
Comment (Tasiobg) [reply composer.json]: I mean, this package is set in composer.json `"symfony/framework-bundle": "^4.1"` why is not `^4.1.0`? This question is not related to this PR changes but I wonder how all the symfony packages fit together, I couldn't find a quick answer.

[the remark this answers is not in the document]
Comment (votemike) [reply composer.json]: I see, not sure why it's set like that. But if this is set, we shouldn't need to update the others.

**Pull request merged**